### PR TITLE
June 30 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 5e7c97b |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-320.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/20230630_amplify.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+There are no new features, bug fixes, or known issues in this release.
+
+### Product Foundry
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Product Foundry
+
+* **Category publication** - Categories owner can publish category only to the Marketplace their team has the publication rights available.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230630_amplify.md:109]: **Category publication** - The owner of a category can make a category visible only in Markeplaces they have publication preferences enabled.

File: content/en/docs/amplify_relnotes/20230630_amplify.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+* **Product resources sort** -  A new option is available when publishing a product: display resource in marketplace grouped per asset. Once this option is activated, the product resources will be sorted first per Asset title and then per resource name in alphabetical order.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230630_amplify.md:110]: **Display resources grouped by asset**-  A new option **display resource in marketplace grouped per asset** is available when publishing a product. Once this option is activated, the resources that appear in the Resource Table of a product in the Marketplace will be displayed sorted in alphabetical order,  first by the Asset title and then by the resource name.

File: content/en/docs/amplify_relnotes/20230630_amplify.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+#### Bug fixes for Product Foundry
+
+* **Product plan creation** - A warning icon is present in beside the plan name just after creating a plan. Now, a refresh action has been added and the warning icon is no moe visible.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230630_amplify.md:114]: **Product plan creation** - After creating a plan, a warning icon was displayed by the plan name. With this fix, a refresh action has been introdicced and the warning icon is no longer visible.

File: content/en/docs/amplify_relnotes/20230630_amplify.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+* **Product visibility side blade** - The publication status of the marketplace is not always accurate. Now, the status is correctly reflected.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230630_amplify.md:115]: **Product visibility** - The publication status of a product in a Marketplace was at times inaccurate. Now, the status is correctly reflected.

File: content/en/docs/amplify_relnotes/20230630_amplify.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+* **Product Category label** - In the product wizard, some extra labels for the Category page have been removed.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230630_amplify.md:116]: **Product Category label** - In the product wizard, some extra labels on the Category page have been removed.

File: content/en/docs/amplify_relnotes/20230630_amplify.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+
+There are no known limitations in this release.
+
+### Business insights
+
+The following enhancements are available in this release.
+
+#### Feature updates for Business insights
+
+* **Multi-value filtering** - All the filters of the Business Insights screens are accepting multiple values as a criteria. And those criteria are persistent from one screen to another hen relevant.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230630_amplify.md:128]: These filtering criteria are persisted from one screen to another when relevant.

File: content/en/docs/amplify_relnotes/20230630_amplify.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+
+## Consumer experience
+
+### Marketplace
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Marketplace
+
+* **Product resources sort** -  A new option is available when publishing a product: display resource in marketplace grouped per asset. Once this option is activated, the product resources will be sorted first per Asset title and then per resource name in alphabetical order.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230630_amplify.md:138]: Display resources grouped by asset - A new option display resource in marketplace grouped per asset is available when publishing a product. Once this option is activated, the resources that appear in the Resource Table of a product in the Marketplace will be displayed sorted in alphabetical order, first by the Asset title and then by the resource name.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Review by robertengaxway — APPROVED** [review]
I approve

**Pull request merged**